src/math/WadRayMath.sol

```diff
@@ -10,12 +10,14 @@ library WadRayMath {
 
     uint256 internal constant WAD = 1e18;
     uint256 internal constant HALF_WAD = 0.5e18;
+    uint256 internal constant WAD_MINUS_ONE = 1e18 - 1;
     uint256 internal constant RAY = 1e27;
     uint256 internal constant HALF_RAY = 0.5e27;
     uint256 internal constant WAD_RAY_RATIO = 1e9;
     uint256 internal constant HALF_WAD_RAY_RATIO = 0.5e9;
     uint256 internal constant MAX_UINT256 = 2**256 - 1; // Not possible to use type(uint256).max in yul.
     uint256 internal constant MAX_UINT256_MINUS_HALF_WAD = 2**256 - 1 - 0.5e18;
+    uint256 internal constant MAX_UINT256_MINUS_WAD_MINUS_ONE = 2**256 - 1 - (1e18 - 1);
```

Comment:
```suggestion
    uint256 internal constant MAX_UINT256_MINUS_WAD_MINUS_ONE = 2**256 - WAD;
```

Reply:
You can't use composed constants in Yul. I'm updating the comment above to make it clear

Reply:
Ok then just `2**256 - 1e18`?

Reply:
That would work too and I'm ok with it, though it's less obvious to the reader

Reply:
Not a blocker we can leave as it is np